Gemfile.jruby-1.9.lock

```diff
@@ -245,10 +245,10 @@ GEM
       logstash-core-plugin-api (>= 1.60, <= 2.99)
       nokogiri
       xml-simple
-    logstash-input-beats (3.1.4)
+    logstash-input-beats (3.1.4-java)
       concurrent-ruby (>= 0.9.2, <= 1.0.0)
       jar-dependencies (~> 0.3.4)
-      logstash-codec-multiline (~> 3.0)
+      logstash-codec-multiline (>= 2.0.5)
```

Comment:
I'm not sure actually. I never hand-edit this file, so not sure how to control it. The gemspec didn't have any changes of this sort.